lib/Agrammon/Web/Routes.pm6

```diff
@@ -485,16 +485,16 @@ sub application-routes(Agrammon::Web::Service $ws) {
         # TODO: test/implement load_branch_data()
```

Comment:
Seems it is implemented now?

Reply:
I'll make a separate PR for that, cleaning up the data sent from the frontend also.